packages/rollups/src/index.ts

```diff
@@ -0,0 +1,101 @@
+import { Rollup } from "@blobscan/db/prisma/enums";
+
+export const ADDRESS_TO_ROLLUP_MAPPINGS: Map<
+  number,
+  Map<string, Rollup>
+> = new Map([
+  // Mainnet
+  [
+    1,
+    new Map([
```

Comment:
Why are you using Map() now?

Reply:
It offers the `entries()` function, which allows you to retrieve both the key and value simultaneously. This makes building inverse mappings from rollup to address more straightforward.